packages/server-core/src/user/user-inventory/user-inventory.hooks.ts

```diff
@@ -10,7 +12,36 @@ export default {
     get: [],
     create: [],
     update: [disallow()],
-    patch: [],
+    patch: [
```

Comment:
we should move this to `user-inventory.class UserInventory.patch` and add the following hooks: `authenticate('jwt')`